README.md

```diff
@@ -0,0 +1,52 @@
+# Web based behavioral closed loop
```

Comment:
This is the current README. I removed references to Firebase but we will need to expand it further when we have the project more fleshed out.